### ISSUE TITLE
Regarding dataset structure

### ISSUE THREAD

**Issue by Raviteja1996** [ywk991112/pytorch-chatbot#17]
Hi I found the opensubtitles dataset from #3 . But according to the README its not in the structure of directory tree.
/data/corpus/open_subtitles
├── opensubtitles_train.txt
├── opensubtitles_valid.txt
└── opensubtitles_test.txt
Its not in the above format. It has some sub folders where each consists of **.xml** and **.info** files. I am new to this please help me if I am missing anything.

**Comment by ywk991112** [comment]
> It has some sub folders where each consists of .xml and .info files.

I don't really know what it means. I'll just try my best to explain to you.
There are some corpus files such as those mentioned in #3, which contains only one file.
You should separate it into three(or more) files. They will be used as training set, validation set, testing set.
Then put all those files under a directory.
Before preprocessing and training, modify the config file first. There are four items(data_dir, train_set, valid_set, test_set) you should modify according to the names of the directory you placed and all those files in it.
